src/MediaPlayer/MediaPlayer.cs

```diff
@@ -119,18 +117,25 @@ public enum MPMediaType : nuint_compat_int {
 		AnyAudio     = 0x00ff,
 		
 		[iOS (5,0)]
+		[Mac (10,12,2)]
```

Comment:
There are a few above this that aren't 10.12.x so we can't hoist here (but I did below).